client/job.lua

```diff
@@ -410,139 +399,109 @@ if config.useTarget then
                 size = vec3(1, 1, 2),
                 rotation = -20,
                 debug = config.debugPoly,
-                onEnter = enteredArmoryZone,
-                onExit = outArmoryZone,
-                inside = insideArmoryZone,
+                onEnter = function()
+                    if QBX.PlayerData.job.onduty then
+                        lib.showTextUI(locale('text.armory_button'))
+                    end
+                end,
+                onExit = function()
+                    lib.hideTextUI()
+                end,
+                inside = function()
+                    OnKeyPress(function()
+                        openArmory(i)
+                    end)
+                end,
             })
         end
 
-        local function enteredRoofZone()
-            if QBX.PlayerData.job.onduty then
-                lib.showTextUI(locale('text.elevator_main'))
-            else
-                lib.showTextUI(locale('error.not_ems'))
-            end
-        end
-
-        local function outRoofZone()
-            lib.hideTextUI()
-        end
-
-        local function insideRoofZone()
-            OnKeyPress(teleportToMainElevator)
-        end
-
         lib.zones.box({
             coords = sharedConfig.locations.roof[1],
             size = vec3(1, 1, 2),
             rotation = -20,
             debug = config.debugPoly,
-            onEnter = enteredRoofZone,
-            onExit = outRoofZone,
-            inside = insideRoofZone,
+            onEnter = function()
+                if QBX.PlayerData.job.onduty then
+                    lib.showTextUI(locale('text.elevator_main'))
+                else
+                    lib.showTextUI(locale('error.not_ems'))
+                end
+            end,
+            onExit = function()
+                lib.hideTextUI()
+            end,
+            inside = function()
+                OnKeyPress(teleportToMainElevator)
+            end,
         })
 
-        local function enteredMainZone()
-            if QBX.PlayerData.job.onduty then
-                lib.showTextUI(locale('text.elevator_roof'))
-            else
-                lib.showTextUI(locale('error.not_ems'))
-            end
-        end
-
-        local function outMainZone()
-            lib.hideTextUI()
-        end
-
-        local function insideMainZone()
-            OnKeyPress(teleportToRoofElevator)
-        end
-
         lib.zones.box({
             coords = sharedConfig.locations.main[1],
             size = vec3(1, 1, 2),
             rotation = -20,
             debug = config.debugPoly,
-            onEnter = enteredMainZone,
-            onExit = outMainZone,
-            inside = insideMainZone,
+            onEnter = function()
+                if QBX.PlayerData.job.onduty then
+                    lib.showTextUI(locale('text.elevator_roof'))
+                else
+                    lib.showTextUI(locale('error.not_ems'))
+                end
+            end,
```

Comment:
```suggestion
            onEnter = function()
                local label = QBX.PlayerData.job.onduty and 'text.elevator_roof' or 'error.not_ems'
                lib.showTextUI(locale(label))
            end,
```
Same thing re: readability